src/App.js

```diff
@@ -97,10 +107,23 @@ class App extends Component {
         if (includeDataPointsGoingForward) {
           const filteredDataPoint = {}
           for (const country of countries) {
-            filteredDataPoint[`newCases${country.value}`] = dataPoint[`newCases${country.value}`] || null
-            filteredDataPoint[`newDeaths${country.value}`] = dataPoint[`newDeaths${country.value}`] || null
-            filteredDataPoint[`totalCases${country.value}`] = dataPoint[`totalCases${country.value}`] || null
-            filteredDataPoint[`totalDeaths${country.value}`] = dataPoint[`totalDeaths${country.value}`] || null
+            const newCases = dataPoint[`newCases${country.value}`] || null
+            const newDeaths = dataPoint[`newDeaths${country.value}`] || null
+            const totalCases = dataPoint[`totalCases${country.value}`] || null
+            const totalDeaths = dataPoint[`totalDeaths${country.value}`] || null
+            filteredDataPoint[`newCases${country.value}`] = newCases
+            filteredDataPoint[`newDeaths${country.value}`] = newDeaths
+            filteredDataPoint[`totalCases${country.value}`] = totalCases
+            filteredDataPoint[`totalDeaths${country.value}`] = totalDeaths
+
+            const popData = dataPoint[`popData${country.value}`] || null
+            console.log(`popdata for ${country.value} is ${popData}`)
+
+            const [nc, nd, tc, td] = [newCases, newDeaths, totalCases, totalDeaths].map((value) => value / popData)
```

Comment:
I think it's more 'standardized' to use per million, so I would just do
```suggestion
            const [nc, nd, tc, td] = [newCases, newDeaths, totalCases, totalDeaths].map((value) => value / popData * 1000000)
```
What do you think?